docs/usage/configuration-options.md

```diff
@@ -243,7 +243,7 @@ Configuring this to `true` means that Renovate will detect and apply the default
 ## branchConcurrentLimit
 
 By default, Renovate won't enforce any concurrent branch limits.
-The `config:base` preset that many extend from limits the amount of concurrent branches to 20, but in most cases we would recommend lower values such as 5 or 10.
+The `config:base` preset that many extend from limits the amount of concurrent branches to 10, but in most cases we would recommend lower values such as 5 or 3.
```

Comment:
Do we really mean for people to put `branchConcurrentLimit` as low as `3`? I imagine setting it to `5` will be low enough already? Is there any need to go lower than `5`?

Where does this number `3` come from anyway? I don't see anything in this PR that explains that number, what am I missing here?

Reply:
Hi @HonkingGoose I used value 3 because the old text was "values as 5 or 10". I replaced it with "values 5 or 3", because 10 is already used by default now. What value do you recommend to use?

Reply:
I see that the head maintainer already made a suggestion that applies to this line. I think this is okay now. Somebody with `write` rights can dismiss my review. 😉